src/App.tsx

```diff
@@ -1,26 +1,104 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
+import { useEffect, useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoCreate } from './components/TodoCreate/TodoCreate';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import * as todoServices from './api/todos';
+import { Todo } from './types/Todo';
+import { StatusFilter } from './types/StatusFilter';
+import { filterTodos } from './utils/filterTodos';
+import { ErrorType } from './types/ErrorType';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [statusFilter, setStatusFilter] = useState<StatusFilter>(
+    StatusFilter.All,
+  );
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.NO_ERROR);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const sendError = (error: ErrorType) => {
+    setErrorType(error);
+
+    const timer = setTimeout(() => {
+      setErrorType(ErrorType.NO_ERROR);
+
+      clearTimeout(timer);
+    }, 3000);
+  };
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const todos = await todoServices.getTodos();
+
+        setTodoList(todos);
+      } catch {
+        sendError(ErrorType.LOAD_TODOS);
+      }
+    })();
+  }, []);
```

Comment:
better or no?what do u think? 🤔 

```suggestion
  useEffect(() => {
    const getTodos = async () => {
      try {
        const todos = await todoServices.getTodos();

        setTodoList(todos);
      } catch {
        sendError(ErrorType.LOAD_TODOS);
      }
    }
    
    getTodos();
  }, []);
```

Reply:
The funny thing is that I used to have this variant in a previous task, but I was told to use IIFE instead. I prefer your suggestion because of its clarity, so I'll go with it in the next task.